plugins/build.gradle

```diff
@@ -11,6 +11,7 @@ allprojects {
     repositories {
         jcenter()
         maven { url 'https://maven.google.com' }
+        maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
@tobrun We should make a note to remove this before final release (ticket?).

Reply:
I consider above comment a great note for that ;)